src/packer.pkr.hcl

```
@@ -124,19 +124,20 @@ build {
   sources = ["source.amazon-ebs.freeipa"]
 
   provisioner "ansible" {
-    extra_arguments = ["--extra-vars", "ansible_python_interpreter=auto"]
-    playbook_file   = "src/upgrade.yml"
+    playbook_file = "src/upgrade.yml"
+    use_sftp      = true
   }
 
   provisioner "ansible" {
-    extra_arguments = ["--extra-vars", "ansible_python_interpreter=auto"]
-    playbook_file   = "src/python.yml"
+    playbook_file = "src/python.yml"
+    use_sftp      = true
```

I saw no indication that this should be removed.

```suggestion
    extra_arguments = ["--extra-vars", "ansible_python_interpreter=auto"]
    playbook_file   = "src/python.yml"
    use_sftp        = true
```

---

Please see [my earlier comment](https://github.com/cisagov/freeipa-server-packer/pull/97#discussion_r1024279288).